.github/workflows/webviz.yml

```diff
@@ -96,6 +96,9 @@ jobs:
       - name: 🤖 Run tests
         working-directory: ./backend
         run: |
+          export WEBVIZ_CLIENT_SECRET="DUMMY_SECRET"
+          export WEBVIZ_SMDA_SUBSCRIPTION_KEY="DUMMY_SUBSCRIPTION_KEY"
+          export WEBVIZ_SMDA_RESOURCE_SCOPE="DUMMY_RESOURCE_SCOPE"
```

Comment:
This is solving the same need as we already do in https://github.com/equinor/webviz/blob/389a69f22f1d9f00db08a8c99ec88b00a4399a51/.github/workflows/webviz.yml#L60.

Since we know need them two places, should we define them as `env` variables higher up in the CI workflow? https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

Reply:
As long as there is no risk of hiding a requirement (which i don't see) I agree.
Will look into it